notification/src/test/java/com/amazon/opendistroforelasticsearch/alerting/destination/CustomWebhookMessageTest.java

```diff
@@ -79,6 +82,49 @@ public void testCustomWebhookMessage() throws Exception {
         assertEquals(expectedCustomWebhookResponse.getStatusCode(), actualCustomResponse.getStatusCode());
     }
 
+    @Test
+    public void testCustomWebhookMessageWithHeaders() throws Exception {
+
```

Comment:
It would be good to assert we are passing the expected header in the HttpPostRequest.